.codespellignore

```diff
@@ -0,0 +1,5 @@
+aks
+witht
```

Comment:
I don't need to ask about "aks," but why are these others in the ignore file? I didn't see any instances of "witht" or "geting" when I did `git grep`.

Reply:
But did you make `git grep -i`? :wink: 

```console
$ codespell -f -H -S .git,_artifacts,*.sum -L aks  | wc -l
149
$ codespell -f -H -S .git,_artifacts,*.sum -L aks  | grep -i witht | head
./azure/scope/machinepoolmachine_test.go:154: WithT ==> with
./azure/scope/machinepoolmachine_test.go:163: WithT ==> with
./azure/scope/machinepoolmachine_test.go:179: WithT ==> with
./azure/scope/machinepoolmachine_test.go:203: WithT ==> with
./azure/scope/machinepoolmachine_test.go:219: WithT ==> with
./azure/scope/machinepoolmachine_test.go:233: WithT ==> with
./azure/scope/machinepoolmachine_test.go:259: WithT ==> with
./azure/scope/machinepool_test.go:104: WithT ==> with
./azure/scope/machinepool_test.go:111: WithT ==> with
./azure/scope/machinepool_test.go:121: WithT ==> with
```

Regarding `aks`, I hit it in almost every single repo I do this, time to submit it upstream I guess :smile:

Reply:
@mboersma since you gave LGTM, I'll mark it as resolved. Thanks again for reviewing!